src/main/resources/i18n/messages.properties

```diff
@@ -341,7 +341,44 @@ project.members.add.error=An error occurred while adding the user to the project
 project.members.add.required=User and project role must be selected before submitting.
 project.members.remove.modal.confirm=You are about to remove <strong>{0}</strong> from the project.
 
-
+# ========================================================================================== #
+# ShareSamples Component                                                                     #
+# ========================================================================================== #
+ShareNoSamples.description=You have no samples selected to share, please go back to the project samples page and select some samples
+ShareNoSamples.link=Go to Project Samples Page
+ShareSamples.title=Share Samples with Another Project
+ShareSamples.projects=Select a project to share samples with
+ShareSamples.ready=These samples are ready to copy
+ShareSamples.exists=Samples that exist in the target project and will not be copied
+ShareSamples.avatar.unlocked=This same is modifiable in this project.
```

Comment:
Change `same` to `sample` on this line and the one below it

Reply:
Fixed in 32d95d2